using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-event-reference.md

```diff
@@ -0,0 +1,178 @@
+
+```text
+{
+    "global.privacy":"optedin",
+    "campaign.timeout":1,
+    "campaign.mcias":"mcias-va7.cloud.adobe.io/mcias",
+    "__dev__campaign.pkey":"",
+    "__stage__campaign.pkey":"",
+    "campaign.pkey":"",
+    "__dev__campaign.server":"mcias.campaign-demo.adobe.com",
+    "__stage__campaign.server":"mcias.campaign-demo.adobe.com",
+    "campaign.server":"mcias.campaign-demo.adobe.com"
+}
+```
+
+### Request Identity  <a id="request-identity"></a>
+
+This event is dispatched by public API to setLinkageFields to set the linkage fields. It is used to download personalized In-App messages.
+
+#### Data payload definition  <a id="data-payload-definition-4"></a>
+
+Here are the key-value pairs in this event:
+
+| **Key** | **Value Type** | **Optional** | **Description** |
+| :--- | :--- | :--- | :--- |
+| `linkage_fields` | Map<String, String> | No | This map contains either one or multiple linkage fields that are used to personally identify the logged in user. |
+
+#### Event data example
+
+```text
+{ 
+    "linkagefields": 
+    { 
+        "firstName":"john", 
+        "lastName":"doe", 
+        "age":30
+    }
+}
+```
+
+### Request Reset  <a id="request-reset"></a>
+
+This event is dispatched when a user decides they no longer wants to utilize personalized In-App messages. Following the recieval of this event, campaign deletes all personalized messages and redownloads only generic messages.
+
+#### Data payload definition  <a id="data-payload-definition-5"></a>
+
+(This event has no event data)
+
+
+### Lifecycle Response Content  <a id="lifecycle-response-content"></a>
+
+This event is a response from the Lifecycle extension to notify a client/extension about lifecycle context data in which the extension/client might be interested. The event is generated by the Lifecycle extension, and the API is called after a lifecycle start, stop, or pause.
+
+#### Data payload definition  <a id="data-payload-definition-6"></a>
+
+Here are the key-value pairs in this event:
+
+| **Key** | **Value Type** | **Optional** | **Description** |
+| :--- | :--- | :--- | :--- |
+| `lifecyclecontextdata` | Map | No ||
+
+#### Event data example
+
+```text
+{    "lifecyclecontextdata": 
+    {
+        "installdate":"22/05/2014",        
+        "hourofday":6,        
+        "dayofweek":2,        
+        "osversion":"iOS 12.3",        
+        "appID":"contextDataValue",        
+        "resolution":"600x1345",        
+        "devicename":"iPhoneX",        
+        "launchevent":"LaunchEvent",        
+        "prevsessionlength":345,        
+        "locale":"en-US",        
+        "runmode":"Application",        
+        "launches":42    
+    },    
+    "starttimestampmillis" :44,    
+    "maxsessionlength":22
+}
+```
+
+
+## Events dispatched
+
+### Campaign Response Content
+This event is a response from the Adobe Campaign Standard extension contains the message interaction details (Viewed, Triggered or Clicked). 
```

Comment:
I think we can add an 'and' to make the sentence a bit clearer: This event is a response from the Adobe Campaign Standard extension and contains the message interaction details (Viewed, Triggered or Clicked).